queue/queue.py

```diff
@@ -1,15 +1,20 @@
 class Queue:
   def __init__(self):
     self.size = 0
-    # what data structure should we
-    # use to store queue elements?
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    self.storage.insert(0, item)
```

Comment:
Other alternative way is to import single linked list and use `add_to_tail` and other pre-built methods. That would require initialization like `self.storage = LinkedList()` assuming we are importing it from the linked list folder.